edit-post/components/header/publish-button/index.js

```diff
@@ -0,0 +1,25 @@
+import { PostPublishPanelToggle, PostPublishButton } from '@wordpress/editor';
+
+const PublishButton = function( {
+	isSidebarOpen,
+	isSidebarEnabled,
+	onToggle,
+	forceIsDirty,
+	forceIsSaving } ) {
```

Comment:
Odd that eslint didn't catch it, but I'd expect `} ) {` on a de-dented newline.

Reply:
7322881e6f257dde3ea4312a43b3f068b0ee3e35